oak/server/rust/oak_runtime/Cargo.toml

```diff
@@ -26,12 +26,17 @@ oak_abi = "=0.1.0"
 prost = "*"
 prost-types = "*"
 rand = "*"
-tokio = { version = "*", features = ["rt-core"] }
+tokio = { version = "*", features = ["rt-core", "macros"] }
 wasmi = { version = "*", default-features = false, features = ["core"] }
+prometheus = { version = "0.8", features = [
+  "nightly"
+], default-features = false }
+futures = "*"
 
 [dev-dependencies]
 maplit = "*"
 wat = "*"
+simple_logger = "*"
```

Comment:
I don't remember if `scripts/format` should sort dependencies in the alphabetical order